portfolio/src/main/webapp/script.js

```diff
@@ -48,3 +48,33 @@ function addPhoto() {
   imageContainer.innerHTML = '';
   imageContainer.appendChild(imgElement);
 }
+
+function getCommentSectionFromServer() {
+  fetch('/comment-section').then(response => response.json()).then((comments) => {
+    const commSection = document.getElementById('comment-list');
+    // Build the comment section.
+    comments.forEach((comm) => {
+      commSection.appendChild(createIndividualComment(comm));
+    });
+  });
+}
+
+// Format each particular comment to show up on page.
+function createIndividualComment(comment) {
+  const liElement = document.createElement('li');
+  // This container is used for showing the username.
```

Comment:
Perhaps just me, though I would have found it slightly more readable if you created and appended the username `<li>` first and then created and appended the comment `<li>`.